scrap.py

```diff
@@ -61,4 +64,5 @@ def scrape_full_post(self) -> dict:
                     else:
                         print("Last post already scraped")
             except exceptions.NotFound as e:
-                raise exceptions.NotFound("Post not found")
+                raise
```

Comment:
Why did you change the error?

Reply:
when an error happens it will raise twice so I did it like this to prevent duplicate

Reply:
It's ok to have two errors it might be you need to change what the error says to reflect what happened for the context